plugins/CorePluginsAdmin/Menu.php

```diff
@@ -65,9 +65,8 @@ public function configureAdminMenu(MenuAdmin $menu)
         }
 
         if ($hasSuperUserAccess) {
-            $menu->addSystemItem(Piwik::translate('General_ManagePlugins') . $pluginsUpdateMessage,
-                $this->urlForAction('plugins', array('activated' => '')),
-                $order = 20);
+            $menu->addPluginItem(Piwik::translate('General_ManagePlugins') . $pluginsUpdateMessage,
+                           $this->urlForAction('plugins', array('activated' => '')), 10);
```

Comment:
```suggestion
                           $this->urlForAction('plugins', ['activated' => '']), 10);
```

Reply:
This PR is not yet marked as ready for review 🙈

Reply:
sorry, too quick 🐎